yo/api_methods.py

```diff
@@ -1,12 +1,27 @@
 # coding=utf-8
 import logging
 
+from yo import transports
+
+from yo import storage
+from yo.storage import users
+
 from jsonrpcserver.aio import methods
 
 logger = logging.getLogger(__name__)
 
+def add_api_method(func):
+    """ Used to add the yo. prefix
+    """
+    methods.add(func,name='yo.%s' % func.__name__)
+
+# test method
+@add_api_method
+async def test(db=None):
```

Comment:
Why not create a health-check method which we can use elsewhere instead of `test`? 
```python
async def test(db=None):
    user_count = yo.storage.users.count_users(db=db)
    return {
         'health':'ok',
         'datetime': datetime.datetime.utcnow().isoformat(),
         'user_count': user_count
    }
```